src/schemas/jwk.ts

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { JTDSchemaType } from 'ajv/dist/core';
```

Comment:
can we import if rom `ajv` and not `ajv/dist/core`

Reply:
It did not work, tests failed with the different import path